spring-cloud-gcp-dependencies/pom.xml

```diff
@@ -65,6 +77,12 @@
 				<version>1.2</version>
 			</dependency>
 
+			<dependency>
+				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-stream</artifactId>
+				<version>1.2.1.RELEASE</version>
```

Comment:
Are you planning to target against this release for any reason? There is an RC version available for `1.3.0` and the GA is scheduled for the end of the month for that.

Reply:
No special reason, we can change to `1.3.0` after it goes GA. I doubt this PR will be ready before end of the month.

Reply:
Noticed some issues with RC1, will wait for the GA version before fully moving to `1.3.0`.